packages/config-plugins/src/utils/Updates.ts

```diff
@@ -19,3 +25,82 @@ export function getUpdateUrl(
   }
   return `https://exp.host/@${user}/${config.slug}`;
 }
+
+export function getNativeVersion(
+  config: Pick<ExpoConfig, 'version'> & {
+    android?: Pick<Android, 'versionCode'> & TempRuntimeVersion;
+    ios?: Pick<IOS, 'buildNumber'> & TempRuntimeVersion;
+  },
+  platform: 'android' | 'ios'
+): string {
+  const version = IOSConfig.Version.getVersion(config);
+  switch (platform) {
+    case 'ios': {
+      const buildNumber = IOSConfig.Version.getBuildNumber(config);
+      return `${version}(${buildNumber})`;
+    }
+    case 'android': {
+      const versionCode = AndroidConfig.Version.getVersionCode(config);
+      return `${version}(${versionCode})`;
+    }
+    default: {
+      throw new Error(
+        `"${platform}" is not a supported platform. Choose either "ios" or "android".`
+      );
+    }
+  }
+}
+
+/**
+ * Compute runtime version policies.
+ * @return an expoConfig with only string valued platform specific runtime versions.
+ */
+export const withRuntimeVersion: ConfigPlugin = config => {
```

Comment:
This will be used here: https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/project/expoConfig.ts in a follow up PR.

Not yet clear how much classic updates will support these new runtime version policies. For now, classic updates expects a string in a top level runtime version, or uses the sdkVersion.